modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/behavior/ListViewBehavior.java

```diff
@@ -262,6 +262,10 @@ public ListViewBehavior(ListView<T> control) {
     private boolean selectionChanging = false;
 
     private final ListChangeListener<Integer> selectedIndicesListener = c -> {
+        if (getNode().getFocusModel() == null) {
+            return;
+        }
+
```

Comment:
line 281 (below): could sm be null there?

Reply:
no, as this is listener is added to a particular selection model.